File: Week2/Tuseday/sum-pairs/js/sumPairs.js
const sumPairs = (array, sum) => {
  let result = []
  for(let i = 0; i <= array.length; i++){
    for(let j = i; j <= array.length; j++){
      if(array[i] + array[j] == sum) {
        result.push([array[i], array[j]])
      }
    }
  }
  if(result.length > 0){
    return result
  } else return 'unable to find pairs'
};

console.log(sumPairs([1,2,3,4,5], 7))

module.exports = {
  sumPairs
}

File: Week2/Tuseday/sum-pairs/python/sum_pairs.py
def sum_pairs(array, pair):
    results = []
    for i in array:
        for j in range(i, len(array)):
            if (i + array[j] == pair):
                results = results + [[i,array[j]]]
    if len(results) > 0:
        return results
    return 'unable to find pairs'